tests/output/ConsoleInteractionTest.py

```diff
@@ -208,16 +269,8 @@ def test_print_result(self):
             diff.delete_line(2)
             diff.change_line(3, "3\n", "3_changed\n")
 
-            with simulate_console_inputs(1), self.assertRaises(ValueError):
-                ApplyPatchAction.is_applicable = staticmethod(
-                    lambda *args: True)
-                print_result(self.console_printer,
-                             self.log_printer,
-                             None,
-                             self.file_diff_dict,
-                             Result("origin", "msg", diffs={
-                                    testfile_path: diff}),
-                             file_dict)
+            ApplyPatchAction.is_applicable = staticmethod(
+                lambda *args: True)
```

Comment:
This basically tested that get_action_info will raise ValueError when passed section == None. Maybe, it's better for this to go into https://github.com/coala-analyzer/coala/pull/2113/commits/803b2af2d0a99e22f81bd0b9eea41eb1a620736f